web/src/components/stageTemplate/add/Input.js

```diff
@@ -50,11 +50,11 @@ const InputSection = props => {
                             component={
                               a.name === 'cmd' ? TextareaField : InputField
                             }
-                            {...a.name === 'cmd' && {
+                            {...(a.name === 'cmd' && {
```

Comment:
This has been included in #1153, please rebase.

Reply:
Already rebased.